Time off tracker WEB/Program.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace Time_off_tracker_WEB
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
```

Comment:
Add Seriloig please, as the implementation in Microsoft.ILogger
take a look at https://metanit.com/sharp/aspnet5/2.10.php and
https://nblumhardt.com/2019/10/serilog-in-aspnetcore-3/

Reply:
I added it. It prints information only to the console for now. I can add a Seq and/or a RollingFile in the logging implementation branch.

Reply:
ok